### PR TITLE
Put output document CSS into a shared stylesheet

### PR DESCRIPTION
**Pull request by stevenwilkin**
Parse it as SCSS

@aduggin

### CODE REVIEW

File: app/templates/output_document.html.erb [hunk 1]
```diff
@@ -3,114 +3,7 @@
...
-      font-size: 18pt;
-      margin-bottom: 20px;
-    }
-
-    .data {
-      font-size: 12pt;
-      font-weight: bold;
-    }
-
+    <%= Sass.compile(File.read(File.join(Rails.root, 'app', 'assets', 'stylesheets', 'output_document.css.scss'))).html_safe %>
```
Comment (andrewgarner) [inline app/templates/output_document.html.erb]: You can make this ever so slightly less offensive by using `Rails.root.join`
```erb
<%= Sass.compile(File.read(Rails.root.join('app', 'assets', 'stylesheets', 'output_document.css.scss'))).html_safe %>
```

Comment (stevenwilkin) [inline app/templates/output_document.html.erb]: Every day a school day

**Comment by stevenwilkin** [comment]
Fixup commit will need rebased

**Comment by andrewgarner** [comment]
Does the job :+1:

**Comment by stevenwilkin** [comment]
Ready for merge @aduggin !

**Comment by gjvis** [comment]
This is now redundant

**Pull request closed without merging**